modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -272,7 +272,7 @@ const DatePicker = ({
 						return onChange('');
 					}
 
-					if (moment(value).isValid()) {
+					if (moment(value, getLocaleDateFormat(locale), true).isValid()) {
```

Comment:
@mlan0, can you please double check if we can remove the parameter `true`?

Reply:
We can't. I tested and it results in a unexpected behavior.

The moment documentation also says something about it in the following example `moment('2016 is a date', 'YYYY-MM-DD').isValid() //returns true, 2016 was matched
`